lib/middleware/validateSession.js

```diff
@@ -0,0 +1,70 @@
+var getSessionVal = function(obj, key) {
+  key = key.split('.');
+
+  for (var i = 0; i < key.length; i++) {
+    var curKey = key[i];
+
+    if (curKey in obj) {
+      obj = obj[curKey];
+    } else {
+      return null;
+    }
+  }
+
+  return obj;
+};
+
+module.exports = function validateSession(mediator, mbaasApi, opts) {
+
+  return function validateSessionMiddleware(req, res, next) {
+    var session,
+      sessionToken;
+
+    opts = {
```

Comment:
@JameelB Yes, it's not work the overhead for this module as it is used in the platform.